state-sync/state-sync-v1/src/fuzzing.rs

```diff
@@ -10,13 +10,12 @@ use crate::{
     shared_components::test_utils,
 };
 use aptos_config::network_id::NetworkId;
-use aptos_infallible::Mutex;
```

Comment:
What's the reason for removing this? 🤔

Reply:
As per the clippy warnings, must use async aware mutex or it may be held on to across an await

Reply:
Crazy